.happy/terraform/modules/ecs-stack/main.tf

```diff
@@ -162,6 +163,38 @@ module upload_batch {
   batch_container_memory_limit = local.batch_container_memory_limit
 }
 
+# TODO: arguments to the upload-cube job go here
```

Comment:
remove?

Reply:
We'll need to register the other variables (namely the WMG bucket). I'll add that to the PR.